admin/admin.php

```diff
@@ -187,7 +207,10 @@ public function meta_box_callback( $post ): void {
 		 *
 		 * @since 1.6.0
 		 */
-		$roles = \apply_filters( 'EmiliaProjects\WP\Comment\notification_roles', $roles );
+		$roles = \apply_filters(
+			'EmiliaProjects\WP\Comment\notification_roles',
+			[ 'contributor', 'author', 'editor', 'administrator', 'super-admin' ]
+		);
```

Comment:
`$roles` was undefined here. The doc for this filter was saying "Defaults to contributor and up", so the array used here contains these roles.